client/app/bundles/comments/components/CommentBox/CommentList/CommentList.jsx

```diff
@@ -52,9 +52,6 @@ export default class CommentList extends BaseComponent {
     const commentNodes = $$comments.map(($$comment, index) => {
       const nodeRef = React.createRef(null);
       return (
-        // `key` is a React-specific concept and is not mandatory for the
-        // purpose of this tutorial. if you're curious, see more here:
-        // http://facebook.github.io/react/docs/multiple-components.html#dynamic-children
         <CSSTransition
           key={$$comment.get('id') || index}
```

Comment:
I am concerned that this is a good key. Though it works in our case I think it is better to make ids like `comment_123`.